test/tests/data/search/a.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>hello</title>
+  </head>
+  <body>
+    <p>mjktkiuewf</p>
```

Comment:
It does look kind of crazy, but I also think it's nice to not need to reset the db. First, it will speed up tests. Second, more non-matching data in the db can only make the test more robust, it seems to me.